.github/workflows/tests_gpu_python.yml

```
@@ -136,7 +136,13 @@ jobs:
       - name: Install required packages
         run: |
           python -m pip install -r requirements-tests.txt
-          python -m pip install "jax[cpu]==0.5.3; python_version<='3.12'" cmake openfermionpyscf
+          python -m pip install cmake openfermionpyscf
+
+      - name: Revert to older JAX version for stable
+        if: inputs.pennylane-version == 'stable'
```

```suggestion
        if: inputs.pennylane-version == 'stable' && inputs.lightning-version == 'stable'
```

I am not sure about this but the problem is only with stable-stable, then I think we need to specify it

---

It does not matter - stable/latest and latest/stable will fail for other reasons

---

and both new and old versions of JAX (0.4.28 or 0.5.3) will cause failures in latest/stable or stable/latest

---

That is true